docs/source/getting_started/projects_and_users.rst

```diff
@@ -0,0 +1,87 @@
+=======================
+Projects and Users
+=======================
+
+Overview
+--------
+
+Projects are containers for other resources, like datasets and modules. Access rights on resources inside a Project are
```

Comment:
```suggestion
Projects are containers for other resources, such as datasets and modules. Access rights on resources inside a Project are
```
Previous wording could be interpreted as saying that projects are like datasets and modules in that they are all containers for other resources